courses/README.md

```diff
@@ -154,8 +154,10 @@ Durations are given in (int)(unit), where units are y, m, d, h or w.
 		defines how many times the student may submit before the personalized exercise is
 		regenerated (the exercise instance is changed to another one). If unset,
 		the exercise is never regenerated.
+	* `model_files`: list of model answers. The model answers are available after the deadline has passed.
```

Comment:
Like I wrote in the discussion, this description could emphasize that the model answers are file paths.

Reply:
Specially, that they are paths, which **do not** start with `/`. Relative paths typically are relative to the file they are in, and not some random location. This is an example of a bad mistake, which can't be fixed.